utils.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
```

Comment:
`pathlib` module is a nice (and seen as more _pythonic_) alternative for managing files, paths, etc.

See this nice RealPython [tutorial](https://realpython.com/python-pathlib/) for more details.